README.md

```diff
@@ -1 +1,31 @@
-# googleapi1
\ No newline at end of file
+Installation & Usage
+
+Installation
+
+Clone or download the repo.
+Open terminal and navigate to Google folder.
+Run npm install to install dependencies.
+
+Usage
+Navigate to server.
+Run nodemon app.js to launch the server.
+Navigate to index.html in client then open with live server.
+
+Tests
+Run npm test to launch test suite(in progress).
+
+Purpose
+An API that can be used to search key words and return a number of premier league teams and their information.
+
+
+Bugs
+[] Testing is not working correctly. Document is not recognised.
+
+Wins & Challenges
+Wins
+Managed to produce an API that successful fetch's information based on user input.
+Completed the task to specifications.
+
+Challenges
+Testing
+
```

Comment:
Unfinished sections. Also could structure better with use of headings and code tags.